src/ai/backend/agent/agent.py

```diff
@@ -1158,6 +1160,33 @@ async def inject_container_lifecycle_event(
             ),
         )
 
+    async def get_free_image_disk(self) -> int | float:
+        """
+        Get free disk for images.
+
+        TODO: get free disk of backend's data root.
+        """
+        stat = os.statvfs("/")
```

Comment:
Let's update here to use the Docker daemon's data root directory, and falls back to "/" if that directory does not exist in the view of the agent.